VideoLocker/src/main/java/org/edx/mobile/util/TextUtils.java

```diff
@@ -0,0 +1,26 @@
+package org.edx.mobile.util;
+
+import android.text.SpannableStringBuilder;
+
+public class TextUtils {
```

Comment:
You should implement a private constructor to prohibit initialization and extension of the utility class.